hhs_oauth_server/request_logging.py

```diff
@@ -245,6 +245,9 @@ def to_dict(self):
             self._log_msg_update_from_dict(
                 request_headers, "req_header_bluebutton_app_version", "X-BLUEBUTTON-APP-VERSION"
             )
+            self._log_msg_update_from_dict(
+                request_headers, "req_header_accept_language", "Accept-Language"
```

Comment:
Is there value in continuing to capture this? My impression from testing is that capturing the `lang` value more reliably captures the language of the page, so we might be okay without this one.

Reply:
yeah this is not important